lib/src/components/NxCombobox/__tests__/NxCombobox.test.tsx

```diff
@@ -0,0 +1,427 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { render } from '@testing-library/react';
+import * as rtlUtils from '../../../__testutils__/rtlUtils';
+
+import userEvent from '@testing-library/user-event';
+
+import NxCombobox, {Props} from '../NxCombobox';
+
+describe('NxCombobox', function() {
+  const minimalProps: Props<string | number> = {
+        value: '',
+        onChange: () => {},
+        onSearch: () => {},
+        matches: []
+      },
+      rtlRender = rtlUtils.rtlRender(NxCombobox, minimalProps);
+
+  //mock scrollIntoView function using jest
+  window.HTMLElement.prototype.scrollIntoView = jest.fn();
+
+  it('renders a div with the nx-combobox class and the specified attributes', function() {
+    const { container } = rtlRender({ title: 'bar' }),
+        component = container.firstElementChild!;
+
+    expect(component.tagName).toBe('DIV');
+    expect(component).toHaveClass('nx-combobox');
+    expect(component).toHaveAttribute('title', 'bar');
+  });
+
+  it('adds the ref to the div', function() {
+    const ref = React.createRef<HTMLDivElement>(),
+        domNode = render(<NxCombobox { ...minimalProps } ref={ref} />).container.firstElementChild!;
+
+    expect(ref.current).toBeDefined();
+    expect(ref.current).toBe(domNode);
+  });
+
+  it('renders an input child with the combobox role', function() {
+    const { getByRole } = rtlRender(),
+        inputElement = getByRole('combobox');
+
+    expect(inputElement.tagName).toBe('INPUT');
+    expect(inputElement).toHaveAttribute('role', 'combobox');
```

Comment:
This is redundant with the fact that you queried by the role in the first place

Reply:
I use `toBeInTheDocument` as suggested below, updated here 4029d58f1778f46cf53cdedad783b595e4a9da4e.